game/scripts/npc/items/item_allied_cyclone.txt

```diff
@@ -100,7 +99,63 @@
         "cast_range_tooltip"                              "575"
       }
     }
-  }
+      "OnSpellStart"
+    {
+
+      "ApplyModifier"
+      {
+        "Target"                                          "TARGET"
+        "ModifierName"                                    "modifier_item_allied_cyclone_active"
+        "Duration"                                        "%allied_cyclone_duration"
+      }
+    }
+    "Modifiers"
+    {
+      "modifier_item_allied_cyclone"
+      {
+        "Passive"                                         "1"
+        "IsHidden"                                        "1"
```

Comment:
Should have `"Attributes"      "MODIFIER_ATTRIBUTE_MULTIPLE"` and `"IsPurgable"      "0"`. Same goes for all the other levels.